curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/resize-opencv.md

```diff
@@ -7,15 +7,14 @@ videoUrl:
 
 ## Description
 <section id='description'>
-<code>p</code> elements are the preferred element for paragraph text on websites. <code>p</code> is short for "paragraph".
-You can create a paragraph element like this:
-<code>&#60;p&#62;I'm a p tag!&#60;/p&#62;</code>
+Scaling is just resizing of the image. OpenCV comes with a function cv.resize() for this purpose. The size of the image can be specified manually, or you can specify the scaling factor.
+We use the function: <code>cv.resize(src, dst, code) </code> where <code>src</code> is an input image,<code>dst</code> - destination one and <code>dsize</code> is a size of output image.
```

Comment:
`cv.resize(src, dst, code)` to `cv.resize(src, dst, dsize)`

Reply:
Please replace the first sentence to something more generic like "Changing images sizes is the most common image processing procedure."